templates/ehub-existing-setup.json

```diff
@@ -0,0 +1,267 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "Name": {
+            "type": "String"
+        },
+        "Storage Name": {
+            "type": "String"
+        },
+        "Alert Logic Access Key ID": {
+            "type": "String"
+        },
+        "Alert Logic Secret Key": {
+            "type": "SecureString"
+        },
+        "Alert Logic API endpoint": {
+            "type": "String",
+            "defaultValue": "api.global-services.global.alertlogic.com",
+            "allowedValues" : [
+                "api.global-services.global.alertlogic.com",
+                "api.global-integration.product.dev.alertlogic.com"
+            ]
+        },
+        "Alert Logic Data Residency": {
+            "type": "String",
+            "defaultValue": "default",
+            "allowedValues": [
+                "default"
+            ]
+        },
+        "Service Principal ID": {
+            "type": "String"
+        },
+        "App Client ID": {
+            "type": "String"
+        },
+        "App Client Secret": {
+            "type": "SecureString"
+        },
+        "Event Hub Resource Group": {
+            "type": "String"
+        },
+        "Event Hub Namespace Name": {
+            "type": "String"
+        },
+        "Event Hub Name": {
+            "type": "String"
+        },
+        "Event Hub Connection String": {
+            "type": "SecureString"
+        },
+        "Event Hub Consumer Group": {
+            "type": "String"
+        }
+    },
+    "variables": {
+        "subscriptionId": "[subscription().subscriptionId]",
```

Comment:
In ehub-new-setup.json we return split(subscription().id, '/')[2] - can this be unified?